aws-xray-recorder-sdk-core/src/main/java/com/amazonaws/xray/entities/EntityImpl.java

```diff
@@ -57,6 +58,7 @@
     @SuppressWarnings("checkstyle:ConstantName")
     @Deprecated
     protected static final ObjectMapper mapper = new ObjectMapper()
+        .registerModule(new JavaTimeModule())
```

Comment:
I suppose we can use the autoFindModule thing that object mapper has to deal with this situation.

Reply:
Can you confirm the values are formatted correctly? I think JavaTimeModule uses ISO format IIRC, which I don't think we want in x-ray model.